src/components/section/completed-section/index.tsx

```diff
@@ -0,0 +1,17 @@
+import classNames from './completed-section.module.scss';
+import Complete from '../../challenges/complete';
+
+const CompletedSection = (props) => {
+  let completeContent;
+
+  if (props.screen == 'Challenges') {
+    completeContent = props.sectionContent.map((challenge) => {
```

Comment:
I think `.filter` is a better candidate. The expectation from `.map` is that you'll get the same number of elements back, that you gave in the original array

Reply:
Ref : https://github.com/Real-Dev-Squad/website-roadmap/pull/8/commits/08fcb4a54dff1dd083ba1c2e4cc8df90ef496daf